app/src/main/java/org/stepic/droid/ui/custom/CoursesCarouselView.kt

```diff
@@ -171,7 +173,7 @@ constructor(
         continueCoursePresenter.detachView(this)
         courseListPresenter.detachView(this)
 
-        ProgressHelper.dismiss(fragmentManager, continueLoadingTag)
+        ProgressHelper.dismiss(activity.supportFragmentManager, LoadingProgressDialogFragment.TAG)
```

Comment:
is it conscious that here `activity.supportFragmentManager` is used here and `fragmentManager` is used in `setBlockingLoading`?